airflow/providers/amazon/aws/operators/emr.py

```diff
@@ -650,7 +650,7 @@ def execute(self, context: 'Context') -> Dict:
                     'jobRunId': response['jobRunId'],
                 },
                 parse_response=['jobRun', 'state'],
-                desired_state=EmrServerlessJobSensor.TERMINAL_STATES,
+                desired_state=EmrServerlessJobSensor.SUCCESS_STATES,
```

Comment:
Do we have test coverage for this?

also Operator imports from Sensor ? :exploding_head:
The statuses should be defined on the Hook

Reply:
Examples:
https://github.com/apache/airflow/blob/9c59831ee78f14de96421c74986933c494407afa/airflow/providers/amazon/aws/hooks/quicksight.py#L39-L40

https://github.com/apache/airflow/blob/6b7a343b25b06ab592f19b7e70843dda2d7e0fdb/airflow/providers/amazon/aws/hooks/emr.py#L178-L194